Standards/scs-0213-v1-k8s-nodes-anti-affinity.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Kubernetes Nodes Anti Affinity
+type: Decision Record
+status: Draft
+track: KaaS
+---
+
+## Introduction
+
+A Kubernetes instance is provided as a cluster, which consists of a set of worker machines,
+so called nodes. A cluster is composed of a control plane and at least one worker node.
+The control plane manages the worker nodes and therefore the pods in the cluster by making
+decisions about scheduling, event detection and global decisions. Inside the control plane,
+multiple components exist, which can be duplicated and distributed over multiple machines
+inside the cluster. Typically, no user containers are run on these machines in order to
+separate the control plane from the live system.
+
```

Comment:
I suggest to add a dedicated glossary section for important terms (e.g. control plane, worker, pods) as in https://github.com/SovereignCloudStack/standards/pull/343.

Reply:
Updated the "Control Plane" part to reflect the K8s docs a bit more.